### PR TITLE
update Install Package page

### PR DESCRIPTION
**Pull request by peterchaudhary**
- The doc is written for "a mother in her mid 40s or 50s" (as discussed with Rafe and Max) as the primary audience. I have filled in the gaps of how exactly to verify a signature.
- Not sure if and where to add Jesse's videos. I don't have the links, but can surely add them in if required.
- New format arranged based on tasks rather than on OS seems to be cleaner
- Brought in and modified the "Build from source" page, since it would technically come under the "Install Wasabi" topic. Plus I'm assuming we are planning to stick with the following outline for Using Wasabi:

1. Install Wasabi
2. Setup wallet
3. Receive bitcoin
4. Send bitcoin
5. Coinjoin
6. Generate wallets
7. Connect hardware-wallet
8. Connect full-node

### CODE REVIEW

File: docs/using-wasabi/InstallPackage.md [hunk 1]
```diff
@@ -5,152 +5,157 @@
...
+    ```
+    tar -pxzf Wasabi-${currentVersion}.tar.gz
+    ./wassabee
+    ```
+:::success Congratulations!
+    Wasabi Wallet should be installed on your computer. You can proceed now to setting up your wallet.
+    :::
+
+## Install GPGTools
+GPGTools makes it breeze to import, verify, and sign digital signatures.
```
Comment (peterchaudhary) [inline docs/using-wasabi/InstallPackage.md:77]: a breeze

File: docs/using-wasabi/InstallPackage.md [hunk 2]
```diff
@@ -5,152 +5,157 @@
...
-![Operating systems supported by Wasabi Wallet](/DownloadAll.png "Operating systems supported by Wasabi Wallet")
-
-Although there is automatic signature verification on Windows and macOS, it is still recommended to manually **VERIFY PGP SIGNATURES** of the downloaded package with zkSNACKs' PGP public key [${zksnacksPublicKeyFingerprint}](https://github.com/zkSNACKs/WalletWasabi/blob/master/PGP.txt) before installing Wasabi.
-This protects you against malicious phishing sites giving you back-doored wallet software.
-If you have personally verified zkSNACKs' PGP public key [${zksnacksPublicKeyFingerprint}](https://github.com/zkSNACKs/WalletWasabi/blob/master/PGP.txt) and you are familiar with the [Web Of Trust](https://security.stackexchange.com/questions/147447/gpg-why-is-my-trusted-key-not-certified-with-a-trusted-signature), please consider also [signing it](https://www.gnupg.org/gph/en/manual/x334.html).
-
-:::tip Always
-Don't trust - Verify!
-:::
+# Install Wasabi
```
Comment (MarnixCroes) [inline docs/using-wasabi/InstallPackage.md:9]: the title should not be renamed (for now)

**Review by MarnixCroes — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/InstallPackage.md (hunk 2 above)
Comment (peterchaudhary) [reply docs/using-wasabi/InstallPackage.md:9]: ok will update this

File: docs/using-wasabi/InstallPackage.md [hunk 3]
```diff
@@ -5,152 +5,157 @@
 }
 ---
 
-# Install package
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:8]:
```suggestion
```

File: docs/using-wasabi/InstallPackage.md [hunk 4]
```diff
@@ -5,152 +5,157 @@
...
-
-![Download Wasabi Wallet for Debian](/DownloadDeb.png "Download Wasabi Wallet for Debian")
-
-3. Verify the signature in the Download repository with `gpg --verify Wasabi-${currentVersion}.deb.asc Wasabi-${currentVersion}.deb`.
-If the message returned says `Good signature from zkSNACKs` and that it was signed with `Primary key fingerprint: ${zksnacksPublicKeyFingerprint}`, then the software was not tampered with since the developer signed it.
-
-	:::tip
-	The output from the verify command may contain `WARNING: This key is not certified with a trusted signature!`.
-	You can ignore this, but if you want to fully verify your download, you need to ask people you trust to confirm that the key fingerprint belongs to zkSNACKs.
+This page contains all the information you will need to safely install the Wasabi Wallet.
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:14]:
```suggestion

This page contains all the information you will need to safely install the Wasabi Wallet. 
```

File: docs/using-wasabi/InstallPackage.md [hunk 5]
```diff
@@ -5,152 +5,157 @@
...
-
-3. Verify the signature in the Download repository with `gpg --verify Wasabi-${currentVersion}.deb.asc Wasabi-${currentVersion}.deb`.
-If the message returned says `Good signature from zkSNACKs` and that it was signed with `Primary key fingerprint: ${zksnacksPublicKeyFingerprint}`, then the software was not tampered with since the developer signed it.
-
-	:::tip
-	The output from the verify command may contain `WARNING: This key is not certified with a trusted signature!`.
-	You can ignore this, but if you want to fully verify your download, you need to ask people you trust to confirm that the key fingerprint belongs to zkSNACKs.
+This page contains all the information you will need to safely install the Wasabi Wallet. 
+It also provides information for developers on how they can [compile from the source code,](#build-from-source) to use the latest features or fixes that have not yet been released.
+Note that the latter section is only meant for experienced users, who know what they are doing, and must be used with extreme caution. For everyone else, the *Install Wasabi Wallet* section is sufficient.
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:16]: Start new sentence with a new line
```suggestion
Note that the latter section is only meant for experienced users, who know what they are doing, and must be used with extreme caution. 
For everyone else, the *Install Wasabi Wallet* section is sufficient.
```

File: docs/using-wasabi/InstallPackage.md [hunk 6]
```diff
@@ -5,152 +5,157 @@
...
+
+## Install Wasabi Wallet
+Follow the steps below to safely install Wasabi Wallet.
+
+1. Download the installer relevant to your operating system (and processor, in case of MacOS) from the [official Wasabi Wallet website.](wasabiwallet.io/#download)
+For improved privacy, we recommended that you use the [Tor browser](https://www.torproject.org/) and download the installer from the [official clearnet-website.](http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion/)
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:22]:
```suggestion
For improved privacy, we recommended that you use the [Tor browser](https://www.torproject.org/) and download the installer from the [Wasabi's official Tor-website.](http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion/)
```

File: docs/using-wasabi/InstallPackage.md [hunk 7]
```diff
@@ -5,152 +5,159 @@
...
+    ```
+    sudo dpkg -i /home/John/Downloads/Wasabi-2.0.1.4.deb
+    ```
+    **MacOS**
+    ```
+    sudo dpkg -i /Users/John/Downloads/Wasabi-2.0.1.4.dmg
+    ```
+    **Windows**
+    ```
+    dpkg -i /Users/John/Downloads/Wasabi-2.0.1.4.msi
```
Comment (peterchaudhary) [inline docs/using-wasabi/InstallPackage.md:67]: not sure if this is accurate @MarnixCroes @yahiheb

Comment (peterchaudhary) [reply docs/using-wasabi/InstallPackage.md:67]: I don't have a Windows machine to verify

File: docs/using-wasabi/InstallPackage.md [hunk 8]
```diff
@@ -5,152 +5,159 @@
...
-This is a version of the software that is thoroughly reviewed by the contributors.
-The package has the binary code that is needed to run the Wasabi Wallet client including the graphical user interface.
-For compiling the open-source code with cutting edge development features, also including the backend server, see this [tutorial here](/using-wasabi/BuildSource.md).
-
-Download the packages either from the official [WasabiWallet.io](https://wasabiwallet.io/) clearnet website or for your privacy's sake, from the official Tor onion service [http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion/](http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion/).
-
-![Operating systems supported by Wasabi Wallet](/DownloadAll.png "Operating systems supported by Wasabi Wallet")
-
-Although there is automatic signature verification on Windows and macOS, it is still recommended to manually **VERIFY PGP SIGNATURES** of the downloaded package with zkSNACKs' PGP public key [${zksnacksPublicKeyFingerprint}](https://github.com/zkSNACKs/WalletWasabi/blob/master/PGP.txt) before installing Wasabi.
```
Comment (peterchaudhary) [inline docs/using-wasabi/InstallPackage.md:23]: If there is automatic signature, but we still recommend manual verification, then we might as well tell users how to manually verify

File: docs/using-wasabi/InstallPackage.md [hunk 9]
```diff
@@ -5,152 +5,161 @@
...
+    **Windows**
+    ```
+    dpkg -i /Users/John/Downloads/Wasabi-2.0.1.4.msi
+    ```
+    **Other Linux distros**
+    ```
+    tar -pxzf Wasabi-${currentVersion}.tar.gz
+    ./wassabee
+    ```
+:::success Congratulations!
```
Comment (peterchaudhary) [inline docs/using-wasabi/InstallPackage.md:76]: not sure if this format is correct

File: docs/using-wasabi/InstallPackage.md [hunk 10]
```diff
@@ -5,152 +5,161 @@
...
+![Download signature](/signature-website.png "Download signature")
+It's a good idea to check the fingerprint on various websites (e.g., mailing lists, discussion forums, social media, personal websites).
+You could text, email, call, video chat, mail, or meet up with people you know to confirm the fingerprint.
+Once you’re certain you've obtained the genuine fingerprint, keep it in a safe place. 
+Every time you need to check whether a key claiming to be from zkSNACKs is authentic, compare that key’s fingerprint to your trusted copy and confirm they match.
+If you have personally [verified](https://gpgtools.tenderapp.com/kb/how-to/trusting-keys-and-why-this-signature-is-not-to-be-trusted#how-to-sign-a-verified-key) zkSNACKs' PGP public key, and you are familiar with the [Web Of Trust](https://www.gnupg.org/gph/en/manual/x334.html), please consider [signing the key.](https://gpgtools.tenderapp.com/kb/how-to/trusting-keys-and-why-this-signature-is-not-to-be-trusted#how-to-sign-a-verified-key)
+
+
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:122]:
```suggestion
```

File: docs/using-wasabi/InstallPackage.md [hunk 11]
```diff
@@ -5,152 +5,161 @@
...
-
-@[youtube](ry6qIyQmzRE)
-
-@[youtube](QHIpEYYqddE)
+This page contains all the information you will need to safely install the Wasabi Wallet. 
+It also provides information for developers on how they can [compile from the source code,](#build-from-source) to use the latest features or fixes that have not yet been released.
+Note that the latter section is meant only for experienced users, who know what they are doing, and must be used with extreme caution.
+For everyone else, the *Install Wasabi Wallet* section is sufficient.
+
+## Install Wasabi Wallet
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:20]:
```suggestion
## Install Wasabi Wallet

```

File: docs/using-wasabi/InstallPackage.md [hunk 12]
```diff
@@ -5,152 +5,161 @@
...
+    **Other Linux distros**
+    ```
+    tar -pxzf Wasabi-${currentVersion}.tar.gz
+    ./wassabee
+    ```
+:::success Congratulations!
+    Wasabi Wallet should be installed on your computer. You can proceed now to setting up your wallet.
+    :::
+
+## Install GPGTools
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:80]:
```suggestion
## Install GPGTools

```

File: docs/using-wasabi/InstallPackage.md [hunk 13]
```diff
@@ -5,152 +5,161 @@
...
+        shasum -a 256 /Users/john/Desktop/gpg/GPG_Suite-2022.1.dmg
+        ```
+        Hit enter to run the command.
+    2. Compare the generated hash with the one provided on the website.
+        If they have match, then the SHA 256 checksum is authentic, and you can go ahead and install the software. 
+        If they don't match, kindly reach out to GPGTools Support team. 
+        ![SHA256 checksum](/gpg-sha.png "Verify SHA256 checksum")
+3. Install GPG. Simply double-click on the installer and follow the directions in the GUI.
+
+## Why verify signature?
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:100]:
```suggestion
## Why verify signature?

```

File: docs/using-wasabi/InstallPackage.md [hunk 14]
```diff
@@ -5,152 +5,161 @@
...
+![Download signature](/signature-website.png "Download signature")
+It's a good idea to check the fingerprint on various websites (e.g., mailing lists, discussion forums, social media, personal websites).
+You could text, email, call, video chat, mail, or meet up with people you know to confirm the fingerprint.
+Once you’re certain you've obtained the genuine fingerprint, keep it in a safe place. 
+Every time you need to check whether a key claiming to be from zkSNACKs is authentic, compare that key’s fingerprint to your trusted copy and confirm they match.
+If you have personally [verified](https://gpgtools.tenderapp.com/kb/how-to/trusting-keys-and-why-this-signature-is-not-to-be-trusted#how-to-sign-a-verified-key) zkSNACKs' PGP public key, and you are familiar with the [Web Of Trust](https://www.gnupg.org/gph/en/manual/x334.html), please consider [signing the key.](https://gpgtools.tenderapp.com/kb/how-to/trusting-keys-and-why-this-signature-is-not-to-be-trusted#how-to-sign-a-verified-key)
+
+
+## Build from source
```
Comment (BTCparadigm) [inline docs/using-wasabi/InstallPackage.md:123]:
```suggestion
## Build from source

```

**Review by BTCparadigm — COMMENTED** [review]
Suggested a bunch of tiny nitpicks

File: docs/using-wasabi/InstallPackage.md [hunk 15]
```diff
@@ -5,152 +5,161 @@
...
+However, if you are downloading the installer from the official website, we highly recommend that you verify the PGP signature of the installer to ensure its authenticity and integrity. 
+One does well to adopt the "distrust the infrastructure" philosophy; for details, see the [Why verify signature?](#why-verify-signature) section.
+
+    The directions below require GPGTools to be installed on your computer. [Install GPGTools](#install-gpgtools) and come back to this section:
+    1. From the [official Wasabi Wallet website](wasabiwallet.io/#download), download the signature of the installer. 
+    To do so, go to the downloads section of the website, and click on **signature** located on the installer relevant to your OS.
+    2. Import zkSNACKs' public key. 
+        1. Open GPGTools (search for "gpg keychain" using spotlight or the menu icon).
+        2. Go to [zkSNACKs' Github repository,](https://github.com/zkSNACKs/WalletWasabi/blob/master/PGP.txt), and copy the raw contents.
+        3. The act of copying, when GPGTools is open, will cause GPGTools to propmt you to decide whether or not to import the key. Accept the request.
```
[the remark this answers is not in the document]
Comment (peterchaudhary) [reply docs/using-wasabi/InstallPackage.md]: modified the syntax for it to read better

**Comment by peterchaudhary** [comment]
> this breaks links
I can change the format; however, there will be a lot of repeated content. In fact, there is not much content to really divide up. So, I'm not quite sure how to rearrange the existing material. Plus, this format is the industry standard for tech docs. The steps usually have headings (Download installer, Verify signature, Install), but since we are not going to be referring to these topics outside of this page, I have removed the headings to make the page really concise.

> BuildSource should NOT be moved to the general install doc. it is advanced and ony used by less than 1% of users AND it breaks links

I can surely remove the `Build from source` section, although for retrievability this seems to be the best place. Also, if you don't mind, could you please confirm the outline of the `Using Wasabi` section, to help me understand where topics such as these would go?

The outline I assumed we are working towards is in the description.

**Comment by MarnixCroes** [comment]
Build from source should not be added to this.

cACK for improving the Install package.
If you plan to continue working on this, please open a new PR

**Pull request closed without merging**